src/store/reducers/dform/initial/styles.json

```diff
@@ -2976,6 +2976,72 @@
             }
           ]
         },
+        {
+          "class": "block",
+          "label": "WIDEL TEST",
+          "children":[
```

Comment:
Both `widelTest.positive` and `widelTest.negative`can be true at the same time as `"class": "check"` is used for these two.
Is it OK? (I'm sorry but I'm not so familiar with it)

Reply:
According to Dr. Ram, if it is positive the more options should be shown and if it is negative the options should not be shown. So, I translated on such form field. If it is negative the data will be stored on CouchDB right? I will talk more about it with Dr. Ram soon and let you know.